app/Http/Controllers/TaskController.php

```diff
@@ -80,6 +104,12 @@ public function update(Request $request, Task $task)
      */
     public function destroy(Task $task)
     {
-        //
+        $is_deleted = DB::table('tasks')
+                        ->where('id', '=', $task->id)
+                        ->delete();
+        if ($is_deleted > 0) {
+
```

Comment:
```suggestion
        if ($is_deleted > 0) {
```
ko có câu lệnh nào khác trong cùng khối lênh ko cần dòng trắng trước return